### DISCUSSION TITLE
LED does not follow the relay state - just blinks on every change

### DISCUSSION THREAD

**Discussion by seamaster101** [RavenSystem/esp-homekit-devices#2325] [MEPLHAA Scripts]
I have simple 1 channel Sonoff module and i run the following script:
{"c":{"io":[[[12,13],2],[[0],6]],"l":13,"b":[[0,5]]},
"a":[{"0":{"r":[[12]]},"1":{"r":[[12,1]]},"b":[[0]]}]}
everything works well, except the LED does not show me when the relay is on. it just blinks on every button press 
How do I make LED turn on when relay is on?

**Comment by RavenSystem** [comment]
LED is used as device status LED. To use as relay status LED, you must manage LED in the same way as relay (but inverted):
```json
{"c":{"io":[[[12,13],2],[[0],6]],"b":[[0,5]]},
"a":[{"0":{"r":[[12],[13,1]]},"1":{"r":[[12,1],[13]]},"b":[[0]]}]}
```

**Comment by seamaster101** [comment]
@RavenSystem thank you for the reply. I did try your suggestion but unfortunately now the LED is on all the time regardless of the relay status. I did copy/paste directly form above, so I think i had it exactly as you suggested. please see the attached video
https://github.com/user-attachments/assets/c273c2fd-f390-4f7a-b32c-57bee7334ad9

**Reply by RavenSystem** [reply]
Sorry. There was a typo. I fixed it. Try again.

**Reply by seamaster101** [reply]
Now it works perfectly well! Thank you!
